raster-foundry/raster-foundry
# Add fields to annotation_label_classes table

## Overview

This PR adds two nullable fields to `annotation_label_classes` table -- `geometry_type` and `description`. It also updates the corresponding data model, dao methods, and property tests.

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new endpoints have scope validation and are included in the integration test csv~

## Testing Instructions

- Make sure the types of these new field make sense to you
- Make sure the migration runs for you
- CI should pass
